sw/device/lib/dif/dif_plic.h

```diff
@@ -227,19 +276,19 @@ dif_plic_result_t dif_plic_irq_claim(const dif_plic_t *plic,
  * Completes the claimed IRQ.
  *
  * Finishes servicing of the claimed IRQ by writing the IRQ source ID back to a
- * target specific CC register. This function must be called after
+ * target specific Claim/Complete register. This function must be called after
  * #dif_plic_irq_claim, when the caller is prepared to service another IRQ with
```

Comment:
Is `#dif_plic_irq_claim` now a prefered method of marking function/variable names over wrapping the name in backticks (`)?

Reply:
I have no idea. That's not my diff. I think we should do a global refactor + announcement if so.

Reply:
You can only use # with global names, you can use backticks with either. We've said use backticks for code, but that isn't necessarily the same as names, and I think people can make their own choices here, rather than mandating x or y.